crates/service_utils/src/db/utils.rs

```diff
@@ -11,9 +11,14 @@ use urlencoding::encode;
 
 pub async fn get_database_url() -> String {
     let db_user: String = get_from_env_unsafe("DB_USER").unwrap();
-    let kms_client = kms::new_client();
-    let db_password_raw = kms::decrypt(kms_client, "DB_PASSWORD").await;
-    let db_password = encode(db_password_raw.as_str()).to_string();
+    let app_env: String = get_from_env_or_default("APP_ENV", "TEST".into());
+    let db_password: String = if app_env.as_str() == "TEST" {
```

Comment:
any reason to deviate from the KMS setup for testing?

Reply:
Why do we do this in development environments? Makes easy setups complex. So I wanted to add in an exception for this. Also, KMS encryption is only for juspay, we should remove this from this repo

Reply:
Sure, make sense.